released/packages/coq-tlc/coq-tlc.20211215/opam

```diff
@@ -0,0 +1,36 @@
+
+opam-version: "2.0"
+maintainer: "arthur.chargueraud@inria.fr"
+
+homepage: "https://github.com/charguer/tlc"
+dev-repo: "git+https://github.com/charguer/tlc.git"
+bug-reports: "https://github.com/charguer/tlc/issues"
+license: "MIT"
+
+synopsis: "TLC: A Library for Classical Coq"
+description: """
+Provides an alternative to the core of the Coq standard library, using classic definitions.
+"""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" { >= "8.13" }
+]
+
+tags: [
+  "category:Miscellaneous/Coq Extensions"
```

Comment:
```suggestion
  "category:Computer Science/Data Types and Data Structures"
```
This is a small nitpick, but we overwhelmingly use the "Coq Extensions" category for plugins. How about the following instead?

Reply:
Fine with me!